config/esp32/components/chip/Kconfig

```diff
@@ -652,20 +652,30 @@ menu "CHIP Device Layer"
 
                 This option is for testing only and should not be enabled in production releases.
 
-        config USE_TEST_PAIRING_CODE
-            string "Use Test Pairing Code"
-            default "CHIPUS"
+        config USE_TEST_SETUP_PIN_CODE
+            string "Use Test Setup Pin Code"
+            range 0 99999999
+            default 12345678
             help
                 Specifies a hard-coded device pairing code to be used if none is found in CHIP NV storage.
-                Setting the value to an empty string disables the feature.
+                Setting the value to 0 disables the feature.
+
+                Note that any decimal integer number between 00000000 and 99999999 can be used here.
+                However, the code within the CHIP commissioner mobile application that orchestrates
+                the device pairing process expects this string to end with a valid Verhoeff check
+                character, and will reject any input that does not.
+
+                This option is for testing only and should not be enabled in production releases.
 
-                Note that any string up to 15 characters can be used here.  However, the code within
-                the CHIP commissioner mobile application that orchestrates the device pairing process
-                expects this string to end with a valid Verhoeff check character, and will reject any
-                input that does not.
+        config USE_TEST_SETUP_DISCRIMINATOR
+            string "Use Test Setup discriminator"
+            range 0x000 0xFFF
+            default 0xF00
+            help
+                Specifies a hard-coded device pairing code to be used if none is found in CHIP NV storage.
```

Comment:
This sentence does not seems to describe the field.

Reply:
Fixed, thanks.